build/packages.targets

```diff
@@ -4,7 +4,7 @@
         <NewtonsoftJsonPackageVersion Condition="$(NewtonsoftJsonPackageVersion) == ''">9.0.1</NewtonsoftJsonPackageVersion>
         <SystemPackagesVersion>4.3.0</SystemPackagesVersion>
         <VSComponentsVersion>16.4.280</VSComponentsVersion>
-        <VSFrameworkVersion>16.5.29714.20</VSFrameworkVersion>
+        <VSFrameworkVersion>16.6.30107.105</VSFrameworkVersion>
```

Comment:
this was already checked into dev by you..
 you may need to rebase your branch to the feature branch...now that the feature branch has this?